docs/source/development/style-guide/review-standards.rst

```diff
@@ -0,0 +1,51 @@
+.. _pull-request-review-standards:
+
+Pull Request and Review Standards
+---------------------------------
+
+Before any code is merged into our code base, it will need to be put up for pull request (PR) and reviewed. The pull request should be created following
+the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>`.
+
+Before opening pull request
+===========================
+
+Before you create the pull request, you should go through the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>` to ensure
+the proposed changes are required and up to our standards.
+
+If you want to work on the pull request or are not yet finished with the code, please indicate this by marking the pull request as
+`wip <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests>`_.
+Anyone looking at the PR will be able to quickly see it is not yet ready for review.
+
+Finally, if you are addressing an existing issue, make sure that issue is linked in your PR. If there is not an existing issue, then you should either create an issue or address WHY you are opening the PR specifically.
```

Comment:
We can all discuss this, but I think if the PR is an implementation for a level 5 requirement an issue needs to be created and linked to the PR. So there's some cases where just opening a PR without an issue and explaining in the PR is fine, but  I think a level 5 implementation or redesign of any kind should be a special case where an issue is required so we can easily keep track of the history of each level 5 by their linked sub-tasks.

Reply:
Ok, I do like that. I will add a note about specific requirements. I'd also be ok with requiring an issue period unless it's a documentation/ github workflow/pre commit PR, but I didn't want to make that change without discussing it with the team